phosphor-auditlog/auditLogMgr.cpp

```diff
@@ -0,0 +1,123 @@
+#include "auditLogMgr.hpp"
+
+#include "auditLogParser.hpp"
+#include "auditLogUtils.hpp"
+
+#include <fcntl.h>
+#include <libaudit.h>
+
+#include <phosphor-logging/lg2.hpp>
+#include <sdeventplus/event.hpp>
+#include <sdeventplus/source/event.hpp>
+#include <xyz/openbmc_project/Common/File/error.hpp>
+
+#include <cstring>
+#include <string>
+
+namespace phosphor
```

Comment:
with newer versions of c++ you can put these on the same line, like:  namespace phosphor::auditlog

Reply:
Thank you, I didn't realize this. I will make this change.